README.md

```diff
@@ -1,5 +1,4 @@
 Gestalt  [![Build Status](https://travis-ci.org/codingteam/Gestalt.png?branch=develop)](https://travis-ci.org/codingteam/Gestalt)
-=======
```

Comment:
On Wed, Apr 22, 2015 at 05:35:21AM -0700, Friedrich von Never wrote:
> > @@ -1,5 +1,4 @@
> >  Gestalt  [![Build Status](https://travis-ci.org/codingteam/Gestalt.png?branch=develop)](https://travis-ci.org/codingteam/Gestalt)
> > -=======
> 
> Why have you removed this? :(

Because I was falling asleep and mistaken it for the merge conflict
marker. Fixed this and other errors you've found, thanks!

> It also seems to generate some trash in `/.rebar/` for me. Should we
> add it here?

Yep, done.

> > # start the service
> > $ ./rel/gestalt/bin/gestalt start
> 
> I have some stupid problems on this step :(
> 
> ```
> > ./rel/gestalt/bin/gestalt start
> D:\X-Files\Projects\Gestalt\rel\gestalt\erts-6.3\bin\erlsrv.exe: The service gestalt_1 is not an erlsrv controlled service.
> ```

SO [suggests](https://stackoverflow.com/questions/17442232/erlang-project-cant-launch-on-windows-generated-by-rebar)
the following workaround:

    $ ./rel/gestalt/bin/gestalt install
    $ ./rel/gestalt/bin/gestalt start

or, if you don't want to install it:

    $ ./rel/gestalt/bin/gestalt console

Please check if this works so that I can update the README.

> I really like all this service stuff, BTW. Is that how Erlang release
> system works? That's great!

[Told you so!](https://github.com/codingteam/Gestalt/issues/9#issuecomment-94174105)

-- 
Regards,
Alexander Batischev